### ISSUE TITLE
Wrong Mail count in the tray. 

### ISSUE THREAD

**Issue by krishnaghatti** [Thomas101/wmail#439]
* Which version of WMail are you using? [e.g. 1.3.1]
2.0.0
* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Mac OSx 10.12.1

I see that the tray icon and the dock icon always show 3 mail count even when there are no unread mail. 
As seen in the screenshot the unread label shows no results.

<img width="611" alt="screen shot 2016-11-19 at 3 05 16 pm" src="https://cloud.githubusercontent.com/assets/10704122/20454544/ef18ccc6-ae69-11e6-8e26-728065c84783.png">

**Comment by Strus** [comment]
To me it seems that when you set message "Done" without reading it then the state of unread mail count is not refreshed.

**Comment by krishnaghatti** [comment]
Well i actually just delete unwanted unread mail. I have not marked them as Done.
As of now even after upgrades, it shows the count 3 always irrespective of how many new mail are available.

**Comment by Thomas101** [comment]
Hi, which setting are you using under Settings > Accounts > Unread Mode?

**Comment by krishnaghatti** [comment]
Ho Thomas,

It is set to ‘All Unread Messages'

Sri Krishna

On Tue, 29 Nov 2016 at 11:43 PM Thomas Beverley

<
mailto:Thomas Beverley <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

Hi, which setting are you using under Settings > Accounts > Unread Mode?

—

You are receiving this because you authored the thread.

Reply to this email directly,
https://github.com/Thomas101/wmail/issues/439#issuecomment-263651459
, or

**Comment by Thomas101** [comment]
If you go to gmail in your web browser and search for `label:inbox label:unread` do you see any messages? It might also be worth trying the [latest pre-release 2.0.7](https://thomas101.github.io/wmail/download) which changes the way it handles notifications and unread counts a bit

**Comment by mindbullet** [comment]
Hi Thomas, I too was unable to clear the unread count. Upgrading from 2.0.0 to 2.0.7 as you suggested worked, well that is, if uninstalling and reinstalling 2.0.0 wouldn't of fixed it (didn't test).

Btw, unread count was there in 2.0.0 despite label:inbox label:unread returning no results but looks like you've solved it with latest pre-release. 

Cheers!

**Comment by krishnaghatti** [comment]
Hi,

There are not unread messages when I check in the browser. 

I tried the pre-release(2.0.7) and it does not show the message count at all. I will try uninstalling completely and re-install.

Sri Krishna

On Wed, 30 Nov 2016 at 12:50 AM Thomas Beverley

<
mailto:Thomas Beverley <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

If you go to gmail in your web browser and search for

label:inbox label:unread

do you see any messages? It might also be worth trying the
https://thomas101.github.io/wmail/download
which changes the way it handles notifications and unread counts a bit

—

You are receiving this because you authored the thread.

Reply to this email directly,
https://github.com/Thomas101/wmail/issues/439#issuecomment-263670240
, or

**Comment by mindbullet** [comment]
Hey Sri,

Your response here showed up in my unread count. When I removed 2.0.0 I used AppCleaner and removed _all_ files associated with wmail 2.0.0

**Comment by krishnaghatti** [comment]
Hi Ross,

Thanks. I will give it a try.

Sri Krishna

On Thu, 01 Dec 2016 at 4:17 PM Ross Innes

<
mailto:Ross Innes <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

Hey Sri,

Your response here showed up in my unread count. When I removed 2.0.0 I used AppCleaner and removed

all

files associated with wmail 2.0.0

—

You are receiving this because you authored the thread.

Reply to this email directly,
https://github.com/Thomas101/wmail/issues/439#issuecomment-264139975
, or
